### DISCUSSION TITLE
Add multiple addtocart classes

### DISCUSSION THREAD

**Discussion by abdullah-shah-5** [EvgeniyMukhamedjanov/liquid-ajax-cart#19] [Q&A]
I want to add two classes one temporary and one for permanent time after adding item to cart 

I tried doing this :     
<script type="application/json" data-ajax-cart-configuration >
  {
    "addToCartCssClass": [ [ "js-my-mini-cart-open", 1500 ] , ["anotherClass"] ] 
  }
</script>

But this gives an error

please help.
Thanks

**Comment by EvgeniyMukhamedjanov** [comment]
There is no such functionality in LAC 
What you can do is use temporary class from data-ajax-cart-configuration
And on top of it add one more snippet to your code that will add another css class after adding to cart:
```html
<script type="application/json" data-ajax-cart-configuration > { "addToCartCssClass":  [ "js-my-mini-cart-open", 1500 ] }
<script type="module">
  import subscribeToCartAjaxRequests from '{{ "liquid-ajax-cart-v1.10.1.js" | asset_url }}';

  subscribeToCartAjaxRequests(( requestState, subscribeToResult ) => {
	subscribeToResult( requestState => {
		if ( requestState.requestType === 'add' && requestState.responseData?.ok ) {
			document.body.classList.add('anotherClass');
		}
	})
  }
</script>
```
